### PR TITLE
Product version number update.

### PR DESCRIPTION
**Pull request by LunfanZhang**
The product version is out of date on Yangtze.

Updating the product major version, product minor version, and product micro version to align with the newest version number.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,6 +1,6 @@
-PRODUCT_MAJOR_VERSION=7
-PRODUCT_MINOR_VERSION=31
-PRODUCT_MICRO_VERSION=0
+PRODUCT_MAJOR_VERSION=8
+PRODUCT_MINOR_VERSION=2
+PRODUCT_MICRO_VERSION=1
```
Comment (stephenchengCloud) [inline Makefile:3]: See comments on https://github.com/xenserver/xe-guest-utilities/pull/157

Comment (LunfanZhang) [reply Makefile:3]: There is different version number with different branch. 8.2.1 is for Yangtze and 8.3.1 is for Next

Comment (LunfanZhang) [reply Makefile:3]: updated

**Review by stephenchengCloud — APPROVED** [review]

**Review by xihuan-citrix — COMMENTED** [review]
https://github.com/xenserver/xe-guest-utilities/commit/f9f379014c70477f013897b643f1f86f197637a2 backport the commit firstly

**Pull request merged**